system-parachains/bridge-hubs/bridge-hub-polkadot/src/bridge_to_ethereum_config.rs

```diff
@@ -15,13 +15,12 @@
 // along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
 
 use crate::{
-	xcm_config,
-	xcm_config::{RelayTreasuryPalletAccount, UniversalLocation},
+	xcm_config::{self, RelayNetwork, RelayTreasuryPalletAccount, UniversalLocation},
 	Balances, EthereumInboundQueue, EthereumOutboundQueue, EthereumSystem, MessageQueue, Runtime,
 	RuntimeEvent, TransactionByteFee,
 };
-pub use bp_bridge_hub_polkadot::snowbridge::EthereumNetwork;
 use bp_bridge_hub_polkadot::snowbridge::{CreateAssetCall, InboundQueuePalletInstance, Parameters};
+pub use bp_bridge_hub_polkadot::snowbridge::{EthereumLocation, EthereumNetwork};
```

Comment:
@yrong @alistair-singh @vgeddes I have compared the [PNA runtime changes](https://github.com/paritytech/polkadot-sdk/pull/5546/files#diff-f639f7c31fb8e9ee5ce9a5b81b1eb1eba5c9a37d05d1fa425d694cf7eec957ecR32) required and all looks correct, from what I can see. Please double check :)